### ISSUE TITLE
Can't use Git w/ GitKraken client

### ISSUE THREAD

**Issue by NickMarley** [keybase/client#8967]
Gives me some error about not found

**Comment by maxtaco** [comment]
Yeah, I noted this, there's nothing we can do, they don't support the standard git external helper protocol.  Please ping them and tell them to do so!

**Comment by maxtaco** [comment]
In the mean time, you can use the `git` CLI for push/clone/fetch/pull, and gitkraken for everything else.

**Comment by NickMarley** [comment]
Ahh. okay. I'll look into using GitHub desktop as your blog post said that is supported.

**Comment by maxtaco** [comment]
GitHub desktop is tested and works

Sourcetree works too...
